lib/routes/charge/sale_view.dart

```diff
@@ -70,31 +75,98 @@ class SaleViewState extends State<SaleView> {
                     iconTheme: Theme.of(context).appBarTheme.iconTheme,
                     textTheme: Theme.of(context).appBarTheme.textTheme,
                     backgroundColor: Theme.of(context).canvasColor,
-                    leading: backBtn.BackButton(iconData: Icons.close),
-                    title: _TotalSaleCharge(
-                      accountModel: accModel,
-                      currentSale: currentSale,
-                      useFiat: widget.useFiat,
-                    ),
+                    leading: backBtn.BackButton(),
+                    title: Text("Current Sale"),
                     actions: <Widget>[
                       IconButton(
                         icon: Icon(
-                          Icons.note_add,
+                          Icons.delete_forever,
                           color: Theme.of(context).iconTheme.color,
                         ),
-                        onPressed: () {},
+                        onPressed: () {
+                          widget.onDeleteSale();
+                        },
                       )
                     ],
                     elevation: 0.0,
                   ),
-                  body: SaleLinesList(
-                      accountModel: accModel, currentSale: currentSale),
-                  bottomNavigationBar: SingleButtonBottomBar(
-                    text: "Clear Sale",
-                    onPressed: () {
-                      posCatalogBloc.actionsSink
-                          .add(SetCurrentSale(Sale(saleLines: [])));
-                    },
+                  extendBody: false,
+                  backgroundColor: Theme.of(context).backgroundColor,
```

Comment:
I believe you were testing primarily on Dark mode. SaleView looks really great on Dark mode but on it's style on blue theme does not match with the rest of our app.
I like the lighter color for list area approach. Please apply it to blue theme with white text/icons on the foreground.

Reply:
The list area is white for the first time and the UI is not final but I think we are at point where @kingonly needs to play with it in order to give feedback. Your feedback here is noted as well.